src/its_preselector/web_relay_preselector.py

```diff
@@ -30,12 +30,18 @@ def set_state(self, i):
             raise Exception("RF path " + key + " configuration does not exist.")
 
     def get_sensor_value(self, sensor_num):
-        sensor_num_string = str(sensor_num)
-        response = requests.get(self.base_url + '?sensor' + sensor_num_string)
+        sensor_num_string =  str(sensor_num)
+        response = requests.get(self.base_url)
         sensor_tag = 'sensor' + sensor_num_string
         root = ET.fromstring(response.text)
         sensor = root.find(sensor_tag)
-        return sensor.text
+        if sensor is None:
+            sensor_tag = 'oneWireSensor' + sensor_num_string
```

Comment:
I've attached pictures of the different formats for the X310 and X410. This begins looking for the X310 xml format looking for a tag like sensor1 and if it is null it will look for the X410 format like oneWireSensor1.
![X310](https://user-images.githubusercontent.com/25595904/171216289-71ba55c0-b459-4326-b46f-3fb02254c557.PNG)
![x410](https://user-images.githubusercontent.com/25595904/171216295-404b3ec9-a41d-4b57-9f79-dbca5a774d8a.PNG)

Reply:
Makes sense. May be good to add a comment to the code mentioning that this is handling a difference in the X310/X410 formats, and that it is prioritizing X310.